tests/conftest.py

```diff
@@ -12,15 +14,28 @@
 )
 
 import jupytext
+from jupytext.cell_reader import rst2md
 from jupytext.cli import system
+from jupytext.myst import is_myst_available
+from jupytext.pandoc import is_pandoc_available
+from jupytext.quarto import is_quarto_available
 
-from .utils import formats_with_support_for_cell_metadata
+from .utils import formats_with_support_for_cell_metadata, tool_version
 
-# Pytest's tmpdir is in /tmp (at least for me), so this helps avoiding interferences between
+# Pytest's tmpdir is in /tmp (at least for me), so this helps to avoid interferences between
 # global configuration on HOME and the test collection
 jupytext.config.JUPYTEXT_CEILING_DIRECTORIES = ["/tmp/"]
 
 
+def isort_version():
+    try:
+        import isort
+
+        return isort.__version__
+    except ImportError:
+        return None
```

Comment:
For another PR, maybe these can be simplified.

Reply:
I have removed that function (it was not used anymore)